src/middlewares/userMiddleware.js

```diff
@@ -0,0 +1,60 @@
+import userValidation from '../validation/index';
+import { Helpers } from '../utils';
+import { UserService } from '../services/index';
+
+const {
+  errorResponse
+} = Helpers;
+const { companySignup } = userValidation;
+
+
+/**
+ * Middleware for input validations
+ */
+export default class ValidationMiddleware {
```

Comment:
Rename the class to match the file name/function
```suggestion
export default class UserMiddleware {
```

Reply:
okay. Thanks @emmanuelnwankwo. I'd fix that in a moment